### PR TITLE
Update into latest and more used version of minecraft.

### PR DESCRIPTION
**Pull request by 9v2**

### CODE REVIEW

File: piemc/server.py
Comment (SixpennyYard) [inline piemc/server.py:1]: You forgot many changes needed to update the server to the standard.

**Comment by andiricum2** [comment]
It is also necessary to change the protocol version to determine the version.

**Comment by 9v2** [comment]
Oops sorry. lol

File: piemc/server.py
Comment (9v2) [reply piemc/server.py:1]: Ok sorry

**Pull request closed without merging**